src/plugins/graphql.js

```diff
@@ -413,25 +437,6 @@ function getVariablesFilter (config) {
   return null
 }
 
-function getSignature (document, operationName, operationType, calculate) {
```

Comment:
I don't think we want to lose what the transforms utility is providing.  Why wouldn't you use another tag at the least?